src/answers-umd.js

```diff
@@ -81,14 +81,8 @@ class Answers {
      * A reference to the formatRichText function.
      * @type {Function}
      */
-    this.formatRichText = (markdown) => {
-      if (typeof markdown !== 'string') {
-        throw new AnswersCoreError(
-          `Rich text "${markdown}" needs to be a string. Currently is a ${typeof markdown}`
-        );
-      }
-      return RtfConverter.toHTML(markdown);
-    };
+    this.formatRichText = (fieldName, markdown) =>
+      RichTextFormatter.format(fieldName, markdown);
```

Comment:
is this meant to be `RichTextFormatter.format(markdown, fieldName);`?

based on https://github.com/yext/answers/pull/785/files#diff-be9a7814200300fa639a80e119e09cd8R24